powershell/generators/nuspec.ts

```diff
@@ -37,10 +37,10 @@ export async function generateNuspec(project: Project) {
     <!-- https://github.com/NuGet/Home/issues/3584 -->
     <file src="${removeCd(project.dll)}" target="${removeCd(project.binFolder)}" />
     <file src="${removeCd(project.binFolder)}/${project.dllName}.deps.json" target="${removeCd(project.binFolder)}" />
-    <file src="${removeCd(project.internalFolder)}/**/*.*" exclude="${removeCd(project.internalFolder)}/readme.md" />
-    <file src="${removeCd(project.customFolder)}/**/*.*" exclude="${removeCd(project.customFolder)}/readme.md;${removeCd(project.customFolder)}/**/*.cs" />
-    <file src="${removeCd(project.docsFolder)}/**/*.md" exclude="${removeCd(project.docsFolder)}/readme.md" />
-    <file src="${removeCd(project.exportsFolder)}/**/ProxyCmdletDefinitions.ps1" />
+    <file src="${removeCd(project.internalFolder)}/**/*.*" exclude="${removeCd(project.internalFolder)}/readme.md" target="${removeCd(project.internalFolder)}" />
```

Comment:
According to https://docs.microsoft.com/en-us/nuget/reference/nuspec#including-assembly-files, we should use "\" instead of "/". Would you try to replace "/" with "\" to verify it works for Mac. If it is OK, please update the PR and I will merge it.

Reply:
> According to https://docs.microsoft.com/en-us/nuget/reference/nuspec#including-assembly-files, we should use "\\" instead of "/". Would you try to replace "/" with "\\" to verify it works for Mac. If it is OK, please update the PR and I will merge it.